extensions/amp-lightbox/0.1/amp-lightbox.js

```diff
@@ -113,6 +88,40 @@ class AmpLightbox extends AMP.BaseElement {
     this.active_ = true;
   }
 
+  build() {
```

Comment:
Let's call it `initialize_()` to be consistent with `amp-img` which takes a similar approach (also note the `_` to indicate private access) 
Also let's add some JS comments indicating this lazily builds the lightbox DOM on the first open ( although we don't always have JS-doc style comments for private methods, it is something we like to do more, but at least we put /** @private */ )